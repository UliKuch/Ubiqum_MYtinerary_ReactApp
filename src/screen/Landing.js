import React from 'react';

function Footer() {
  return (
    <footer className="footer mt-5">
      <div className="d-flex justify-content-center container" style={{ height: 30 }}>

        {/* TODO: insert link to home */}

        <a className="h-100" href="#"><img className="h-100" src="./images/homeIcon.png" alt="A clickable home button"/></a>
      </div>
    </footer>
  )
}


function LandingMain() {
  return (
    <div className="d-flex flex-column align-items-center">
      <p style={{ textAlign: "center" }}>Find your perfect trip, designed by insiders who know and love their cities.</p>
      <h3>Start browsing</h3>
      <div className="mb-5" style={{ height: 100 }}>

        {/* TODO: insert link to cities page */}
        
        <a className="h-100" href=""><img className="h-100" src="./images/circled-right-2.png" alt="A clickable button displaying an arrow."/></a>
      </div>
      <p>Want to build your own MYtinerary?</p>
      <div className="d-flex justify-content-around w-100">
        
        {/* TODO insert links to login page and to create account page */}
        
        <a href="">Login</a>
        <a href="">Create Account</a>
      </div>
    </div>
  )
}


function Logo() {
  return (
    <div className="d-flex justify-content-center mb-5" style={{ height: 100 }}>
      <img className="h-100" src="./images/MYtineraryLogo.png" alt="The MYtinerary Logo"/>
    </div>
  )
}


function Landing() {
  return (
   <div>
    <Logo />
    <LandingMain />
    <Footer />
   </div>
  )
}


export default Landing;