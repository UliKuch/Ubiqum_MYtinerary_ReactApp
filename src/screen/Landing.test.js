import React from 'react';
import { shallow } from 'enzyme';
import Landing from './Landing';
import toJson from 'enzyme-to-json';


it('Landing component should render without crashing', () => {
    shallow(<Landing />);
});

it('Landing component snapshot', () => {
    const wrapper = shallow(<Landing />);

    expect(toJson(wrapper)).toMatchSnapshot();
});