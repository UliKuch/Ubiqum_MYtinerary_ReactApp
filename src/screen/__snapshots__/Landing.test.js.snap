// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing component snapshot 1`] = `
"<div>
  <Logo />
  <LandingMain />
  <Footer />
</div>"
`;
